lms/djangoapps/grades/signals/signals.py

```diff
@@ -78,3 +78,8 @@
         'subsection_grade',  # SubsectionGrade object
     ]
 )
+
+
+# Signal that indicates that a user's grade for a course has been updated.
+# This is a downstream signal of SUBSECTION_SCORE_CHANGED.
+from openedx.core.djangoapps.signals.signals import COURSE_GRADE_CHANGED
```

Comment:
I deliberately put it here for 2 reasons:
1. This signal should really be in this file, along with the other ones.  However, it was resulting in a bigger change - because of circular dependencies.
2. I wanted to keep the ordered list of signals from problem-scoped to course-scoped.